autospec/build.py

```diff
@@ -203,7 +203,7 @@ def get_mock_cmd():
     return 'sudo /usr/bin/mock'
 
 
-def package(filemanager, cleanup=False):
+def package(filemanager, cleanup=False, mockconfig=False):
```

Comment:
You set mockconfig to be the bool `false` but then treat it as a string below. Please set this to be `""` by default instead.